doit/dependency.py

```diff
@@ -693,6 +693,12 @@ def get_status(self, task, tasks_dict, get_log=False):
 class UptodateCalculator(object):
     """Base class for 'uptodate' that need access to all tasks
     """
+    
```

Comment:
Uhmm. I don't like adding this everywhere without understanding exactly why it was necessary. Do you have the full traceback of the failure without this patch (including the pickle debug information)?